bin/MadGraph5_aMCatNLO/cards/production/13TeV/darkphotonDimuon/darkphotonDimuon_DPMass5e-1MeV_DF1e-2/darkphotonDimuon_DPMass5e-1MeV_DF1e-2_proc_card.dat

```diff
@@ -0,0 +1,10 @@
+import model --modelname  HAHM_MG5model_v3/HAHM_variableMW_v3_UFO
```

Comment:
If you remove the --modelname option, I think this would remove the m+/mu+ problem

Reply:
if I remove --modelname, madgraph also complains:
The import format was not given, so we guess it as command
import model HAHM_MG5model_v3/HAHM_variableMW_v3_UFO
INFO: failed to load model from pickle file. Try importing UFO from File
Command "import /uscms_data/d3/yyao/DarkPhoton_2/Gridpack2/genproductions/bin/MadGraph5_aMCatNLO/darkphotonDimuon_DPMass5e-1MeV_DF1e-2/darkphotonDimuon_DPMass5e-1MeV_DF1e-2_gridpack/work/darkphotonDimuon_DPMass5e-1MeV_DF1e-2_proc_card.dat" interrupted in sub-command:
"define higgs = h hs" with error:
InvalidCmd : No particle hs in model

Seems that madgraph is not able to load the model without that option. I am not sure how to make it work. Let me know if you have an idea.

Reply:
That just indicates the naming conventions of MadGraph for "hs" is different from the model card you are using.

@agrohsje Should we ask to check all the subsequent changes to remove l+ = e+ m+ lines or is it okay if we just keep it this way?

Reply:
Hi @sihyunjeon @agrohsje For this question, I can change hs to h2 to make it work, but to be honest it doesn't really matter. Let me know which way you would prefer. I am pretty sure m+ is mu+, it is just a convention issue. I found something similar from Madgraph forum, and Olivier answered there:

"what is the output of "import model YOUR_MODEL_NAME"
It should display the name of all particles as one of the last line.

Sometimes (but I do not think that this is your issue),
you might one to use this command instead:
import model YOUR_MODEL_NAME --modelname

the --modelname forces to follow the UFO convention for the model instead of using
MG5aMC convention for the name of the particles (note that you can change that MG5 convention by editing the input/particles_name_default.txt
As said, I do not think that this is your issue.

Cheers,

Olivier"

Reply:
I'll let Alexander to decide. Either way is fine for me

Reply:
Fine for me.